### PR TITLE
Add Workbench filter to search by tmTextUnitId

### PR DESCRIPTION
**Pull request by garionpin**
Added a new "TM Text Unit ID" option in the Workbench filter dropdown
to be able to search by the IDs of the text units.

### CODE REVIEW

File: webapp/src/main/resources/properties/de.properties [hunk 1]
```diff
@@ -142,6 +142,9 @@ search.filter.asset=Asset
 # Search filter option "plural form"
 search.filter.pluralFormOther=Der Plural von
 
+# Search filter option "tm text unit ID"
+search.filter.tmTextUnitId=Tm Text Unit ID
```
Comment (aurambaj) [inline webapp/src/main/resources/properties/de.properties:146]: localized files are generated, no need to edit them

**Review by aurambaj — COMMENTED** [review]
Looks good! The deep linking/reload for URL was already implemented, right?

File: webapp/src/main/resources/properties/de.properties (hunk 1 above)
Comment (garionpin) [reply webapp/src/main/resources/properties/de.properties]: I've removed the non En files.

**Comment by garionpin** [comment]
> Looks good! The deep linking/reload for URL was already implemented, right?

Yes, that was already implemented - I did however test the functionality again as part of these changes to make sure everything still works for both scenarios.

**Pull request merged**